### DISCUSSION TITLE
Points not awarded

### DISCUSSION THREAD

**Discussion by owldragonkc** [codewars/codewars.com#2683] [Q&A] [answered]
I notice I (username: owldragon) completed some katas and got some progress, but later my progress was removed. My rank score is 2504 (https://www.codewars.com/api/v1/users/owldragon), but I calculated that it should be 2763, based on the katas I've solved. Is there a problem with my account or did I do something wrong? Is it because I hardcoded an answer? Thanks for your help.

**Comment by Blind4Basics** [comment, marked as answer]
2763 it is, now

**Reply by owldragonkc** [reply]
Thanks for fixing that!
